app/src/main/res/values/strings.xml

```diff
@@ -95,6 +95,28 @@
     <string name="pref_play_vibration_after_rest_timer">Play vibration</string>
     <string name="pref_play_vibration_after_rest_timer_summary">Play vibration after rest timer finished</string>
 
+    <string name="pref_scheduled_backup_setting">Scheduled Backup Settings</string>
```

Comment:
I would rather put "Automatic Backup Settings". 
"Scheduled" would rather mean that you can set a specific date.

Reply:
Solved by 8d7006ef351c58d01ce4e23dcfeb8413e5987de0.